resources/js/Components/Country.vue

```diff
@@ -0,0 +1,74 @@
+<script setup>
+import { ref } from 'vue'
+import { useForm } from '@inertiajs/vue3'
+
+const deleteCountryForm = useForm({})
+
+const props = defineProps({
+  country: Object,
+})
+
+function destroyCountry(countryId) {
+  deleteCountryForm.delete('/countries/' + countryId)
+}
+
+function updateCountry(countryId) {
+  updateCountryForm.put('/countries/' + countryId)
```

Comment:
```suggestion
  updateCountryForm.patch(`/countries/${countryId}`)
```
**Patch** should be the preferred method here, since we are updating the data of an existing object.